docs/lessons/images.rst

```diff
@@ -0,0 +1,234 @@
+Images
+------
+
+MicroPython is about as good at art as you can be if the only thing you have is
+a 5x5 grid of red LEDs (light emitting diodes - the things that light up on the
+front of the device). MicroPython gives you quite a lot of control over the
+display so you can create all sorts of interesting effects.
+
+MicroPython comes with lots of built-in pictures to show on the display.
+For example, to make the device appear happy you type::
+
+    from microbit import *
+    display.show(Image.HAPPY)
+
+I suspect you can remember what the first line does. The second line uses the
+``display`` object to ``show`` a built-in image. The happy image we want to
+display is a part of the ``Image`` object and called ``HAPPY``. We tell
+``show`` to use it by putting it between the parenthesis (``(`` and ``)``).
+
+.. image:: happy.png
+
+Here's a list of the built-in images:
+
+    * ``Image.HEART``
+    * ``Image.HEART_SMALL``
+    * ``Image.HAPPY``
+    * ``Image.SMILE``
+    * ``Image.SAD``
+    * ``Image.CONFUSED``
+    * ``Image.ANGRY``
+    * ``Image.ASLEEP``
+    * ``Image.SURPRISED``
+    * ``Image.SILLY``
+    * ``Image.FABULOUS``
+    * ``Image.MEH``
+    * ``Image.YES``
+    * ``Image.NO``
+    * ``Image.CLOCK12``, ``Image.CLOCK11``, ``Image.CLOCK10``, ``Image.CLOCK9``,
+      ``Image.CLOCK8``, ``Image.CLOCK7``, ``Image.CLOCK6``, ``Image.CLOCK5``,
+      ``Image.CLOCK4``, ``Image.CLOCK3``, ``Image.CLOCK2``, ``Image.CLOCK1``
+    * ``Image.ARROW_N``, ``Image.ARROW_NE``, ``Image.ARROW_E``,
+      ``Image.ARROW_SE``, ``Image.ARROW_S``, ``Image.ARROW_SW``,
+      ``Image.ARROW_W``, ``Image.ARROW_NW``
+    * ``Image.TRIANGLE``
+    * ``Image.TRIANGLE_LEFT``
+    * ``Image.CHESSBOARD``
+    * ``Image.DIAMOND``
+    * ``Image.DIAMOND_SMALL``
+    * ``Image.SQUARE``
+    * ``Image.SQUARE_SMALL``
+    * ``Image.RABBIT``
+    * ``Image.COW``
+    * ``Image.MUSIC_CROTCHET``
+    * ``Image.MUSIC_QUAVER``
+    * ``Image.MUSIC_QUAVERS``
+    * ``Image.PITCHFORK``
+    * ``Image.XMAS``
+    * ``Image.PACMAN``
+    * ``Image.TARGET``
+    * ``Image.TSHIRT``
+    * ``Image.ROLLERSKATE``
+    * ``Image.DUCK``
+    * ``Image.HOUSE``
+    * ``Image.TORTOISE``
+    * ``Image.BUTTERFLY``
+    * ``Image.STICKFIGURE``
+    * ``Image.GHOST``
+    * ``Image.SWORD``
+    * ``Image.GIRAFFE``
+    * ``Image.SKULL``
+    * ``Image.UMBRELLA``
+    * ``Image.SNAKE``
+
+There's quite a lot! Why not modify the code that makes the micro:bit look
+happy to see what some of the other built-in images look like? (Just replace
+``Image.HAPPY`` with one of the built-in images listed above.)
+
+DIY Images
+++++++++++
+
+Of course, you want to make your own image to display on the micro:bit, right?
+
+That's easy.
+
+Each LED pixel on the physical display can be set to one of ten values. If a
+pixel is set to ``0`` (zero) then it's off. It literally has zero brightness.
+However, if it is set to ``9`` then it is at its brightest level. The values
```

Comment:
`if it is set to ``9`` then it is at its brightest level` to
`if it is set to ``9`` then it is at its highest brightest level`?